integration_tests/run.sh

```diff
@@ -1,37 +1,56 @@
 #!/bin/bash
 # Script for jenkins to run tests on polar2grid.
 
-set -exf
+set -ex
 export PATH="/usr/local/texlive/2019/bin/x86_64-linux":$PATH
 cd "$WORKSPACE"
 # Activate conda for bash.
 /data/users/davidh/miniconda3/bin/conda init bash
 # Restart the shell to enable conda.
 source ~/.bashrc
-conda env update -n jenkins_p2g_env -f build_environment.yml --prune
+conda env update -n jenkins_p2g_env -f build_environment.yml
 conda activate jenkins_p2g_env
 
-time=`date +"%Y%m%d-%H%M%S"`
-tarball_name=polar2grid-swbundle-$time
-./create_conda_software_bundle.sh $WORKSPACE/$tarball_name
-export POLAR2GRID_HOME="$WORKSPACE/$tarball_name"
+# Handle release vs test naming.
+prefix="$(cut -d'-' -f1 <<<"$GIT_TAG_NAME")"
+end="`date +"%Y%m%d-%H%M%S"`"
+# If the tag is correct and a version was specified, make a version release.
+if [[ "$GIT_TAG_NAME" =~ [pg]2g-v[0-9]+\.[0-9]+\.[0-9]* ]]; then
+    # Removes prefix from $GIT_TAG_NAME.
+    end=${GIT_TAG_NAME#"$prefix-"}
+fi
+if [[ "$prefix" = "g2g" ]]; then
+    prefix=geo
+else
+    prefix=polar
+fi
+swbundle_name=${prefix}2grid-swbundle-${end}
+
+./create_conda_software_bundle.sh ${WORKSPACE}/${swbundle_name}
+export POLAR2GRID_HOME="$WORKSPACE/$swbundle_name"
+conda env update -n jenkins_p2g_env -f jenkins_environment.yml
 cd "$WORKSPACE/integration_tests"
 behave --no-logcapture --no-color --no-capture -D datapath=/data/users/kkolman/integration_tests/polar2grid/integration_tests/p2g_test_data
 
 # Only ran by Jenkins if build was successful.
-# Save software bundle.
-rm -rf /tmp/polar2grid-*
-mkdir /tmp/polar2grid-$time
-cp -r "$WORKSPACE/$tarball_name" /tmp/polar2grid-$time
+# Remove old software bundles.
+rm -rf /tmp/${prefix}2grid-*
+mkdir /tmp/${prefix}2grid-${end}
+# Save software bundle and tarball.
+cp -r "$WORKSPACE/$swbundle_name" /tmp/${prefix}2grid-${end}
```

Comment:
If you are copying the tarball I'm not sure you need to copy the extracted directory. Thoughts?

Reply:
I'm okay with that, I just saw that both of each version were in data/dist, and if they get unpacked anyways why not just put both the tarball and unpacked copies in data/dist.

Reply:
The tarballs won't always be unpacked. If we prove that they work there is a good chance Kathy and I can just take them and give them to people. Or Kathy might download them to another server and unpack them there. Overall, it just limits how much disk space is used.